### PR TITLE
Re-adds Warm Donk-Pockets

### PR DESCRIPTION
**Pull request by Iamgoofball**
bsqo9z1

that old code was such shit

so i improved it

now only 15 lines instead of 37

for those not in the know about the heated pockets of lore:

you can microwave donk pockets to get some tricord added to them

### CODE REVIEW

**Comment by paprka** [comment]
yeah whatever

**Comment by Iamgoofball** [comment]
that's for a different pull request

**Comment by ike709** [comment]
:+1:  x :100:

**Comment by Fox-McCloud** [comment]
:+1:

**Comment by AnonymousNow** [comment]
Praise Jebus.

**Comment by MrPerson** [comment]
You can still put infinite tricord in a single donk pocket, meaning you never actually fixed the original problem

**Comment by optimumtact** [comment]
If the microwave spawns them via new, surely they would all only have the starting 5 tricord.

It's not copying the reagents is it?

**Comment by Miauw62** [comment]
i dont think you can, mrp.

File: code/modules/food&drinks/recipes/microwave/recipes_pastry.dm [hunk 1]
```diff
@@ -75,6 +75,13 @@
 	)
 	result = /obj/item/weapon/reagent_containers/food/snacks/donkpocket
 
+/datum/recipe/donkpocket/warm
+	reagents = list()
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/donkpocket
```
Comment (Malkevin) [inline code/modules/food&drinks/recipes/microwave/recipes_pastry.dm]: Have you checked that you can't remicrowave warm pockets?

**Comment by Iamgoofball** [comment]
@MrPerson it'd be helpful if you could explain exactly how you do this so I can fix that bug if it's even present in this version

**Comment by Iamgoofball** [comment]
alright so just to be certain I made sure that duplication of tricord shouldn't be able to happen unless there's a method I don't know about

**Comment by Jalleo** [comment]
Have you tested it with the upgraded microwave?

**Comment by Iamgoofball** [comment]
you can upgrade the microwave?

**Comment by tkdrg** [comment]
Yes, and that's the entire reason they were removed in the first place. Upgraded microwave duplicates donk pockets, thus creating tricord from nothing.

**Comment by Miauw62** [comment]
it already generates tricord from nothing

**Comment by ExplosiveCrate** [comment]
The issue was that you could create infinite donks because they cooled down and could be re-added to the microwave.

Of course, that should that should be impossible now.

**Comment by Malkevin** [comment]
You didn't even need to wait for them to cool, the microwaves never checked for it

**Comment by Jordie0608** [comment]
Changelog

**Comment by Cheridan** [comment]
These are obj/item/weapon/reagent_containers/food/snacks/donkpocketwarm, not obj/item/weapon/reagent_containers/food/snacks/donkpocket/warm. So the recursive donk pockets shouldn't work.

**Comment by Jalleo** [comment]
@Iamgoofball changelog required for this to be merged

**Comment by Fox-McCloud** [comment]
@Iamgoofball 

needs to be omnizine now.

**Comment by Iamgoofball** [comment]
Uh oh

**Pull request merged**